interface/logKappa.h

```diff
@@ -0,0 +1,23 @@
+#ifndef logKappa_h
+#define logKappa_h
+
+template<typename T>
+T logKappa(const T x, const double &kappaHigh, const double &kappaLow) {
```

Comment:
Given that what you are passing here are `log(kappaHigh)` and `log(kappaLow)`, I think these variable names could be updated to something that matches better what they represent?